deployments/curity/apigateway/openresty/default.conf.template

```diff
@@ -0,0 +1,91 @@
+#
+# Use the Docker DNS server and create a cache for the phantom token plugin
+#
+resolver 127.0.0.11;
+lua_shared_dict phantom-token  10m;
+
+#
+# The route used by the SPA to get its static content
+#
+server {
+    server_name www.product.example;
+    listen      3001;
```

Comment:
why the port?

Reply:
You have to provide one. We use a low privilege port above 1024 in this type of example. I used the same one as we used for Kong. Note that the SPA (if run externally in development mode) uses port 3000, so I used 3001 here to avoid conflicts.